pycon_archive_past_website/common/dataio.py

```diff
@@ -0,0 +1,41 @@
+"""Data related functions"""
+import os
+from urllib.parse import unquote, urlparse
+
+from loguru import logger
+
+
+def mkdir(path: str):
+    """
+    Create folder from execute entry folder
+
+    Args:
+        path (str): relative path
+    """
+    path = urlparse(path).path
+    try:
+        # correct the path to directory path and be a local path
+        dir = "." + path[0 : path.rfind("/") + 1]
```

Comment:
nit: using `pathlib.Path` here would get more expressive. I'm not quite sure if `"."` here is redundant, could you help me check this?
```suggestion
        dir = ("." / Path(path).parent).resolve()
```

Reply:
Updated.